pkgs/by-name/gi/git-buildpackage/package.nix

```diff
@@ -0,0 +1,83 @@
+{
+  coreutils,
+  debian-devscripts,
+  dpkg,
+  fetchFromGitHub,
+  git,
+  lib,
+  man,
+  python3,
+  stdenv,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "git-buildpackage";
+  version = "0.9.37";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "agx";
+    repo = "git-buildpackage";
+    tag = "debian/${version}";
+    hash = "sha256-0gfryd1GrVfL11u/IrtLSJAABRsTpFfPOGxWfVdYtgE=";
+    fetchSubmodules = true;
+  };
+
+  postPatch = ''
+    substituteInPlace gbp/command_wrappers.py \
+    --replace-fail "/bin/true" "${lib.getExe' coreutils "true"}" \
+    --replace-fail "/bin/false" "${lib.getExe' coreutils "false"}"
+  '';
+
+  build-system = [
+    python3.pkgs.setuptools
+  ];
+
+  dependencies = with python3.pkgs; [
+    python-dateutil
+  ];
+
+  pythonImportsCheck = [
+    "gbp"
+  ];
+
+  nativeCheckInputs = with python3.pkgs; [
+    coverage
+    debian-devscripts
+    dpkg
+    git
+    pytest-cov
+    pytestCheckHook
+    man
+    pyyaml
+    rpm
+  ];
+
+  preCheck = ''
+    export HOME=$(mktemp -d)
```

Comment:
```suggestion

```
This should be `writableTmpDirAsHomeHook` in checkInputs

Reply:
Thanks, fixed